modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -297,17 +287,6 @@ def get_clinic_memoized(location_id, clinic_id)
       end
       memoize :get_clinic_memoized
 
-      def get_facility_memoized(location_id)
```

Comment:
I also relocated this function as `#get_facility_timezone` depended on it as well. The appointments service already had `get_facility` so I'm not sure why the memoized version wasn't already in there as well